developing_features/guides/new_project.md

```diff
@@ -0,0 +1,44 @@
+# [Developing Features](../developing_features.md) / Guides / Setting Up a New Project
+
+
+#### Checklist
+
+###### For a Public project
+
+ - Generate a new gem with `bundle gem abc`
+ - Commit it, before changing anything, as `[skip] Initial commit from Bundler 1.16.1 (1m)` (giving the appropriate version of Bundler)
+ - [Create a new repo on GitHub](https://github.com/new) (e.g. `cph/abc`)
```

Comment:
TODO: add steps for configuring **Collaborators & teams**

Reply:
<img width="748" alt="screen shot 2019-01-11 at 8 39 28 am" src="https://user-images.githubusercontent.com/151907/51045589-9110a200-1589-11e9-86a2-d95772e45c3a.png">